benchexec/tools/ceagle.py

```diff
@@ -24,19 +25,8 @@
 
 class Tool(benchexec.tools.template.BaseTool):
 
-    REQUIRED_PATHS = [
-                  "ceagle.sh",
-                  "dfs.py",
-                  "parse2str.py",
-                  "parsece.sh",
-                  "svcore",
-                  "svie",
-                  "verifier.py",
-                  "z3"
-                  ]
-
     def executable(self):
-        return util.find_executable('ceagle.sh')
+        return util.find_executable('sv-ceagle')
```

Comment:
Changing the executable name will make it impossible to run previous versions of Ceagle with a newer BenchExec, which is unfortunate if people want to reproduce old results.

Is this change really necessary? Do you have a consistent entry point to your tool that is stable over the years?